src/test/java/com/sonyericsson/jenkins/plugins/bfa/PluginImplHudsonTest.java

```diff
@@ -153,48 +104,21 @@ public void testConfigureLowScanThreads() throws Exception {
      */
     @Test
     public void testConfigureConvert() throws Exception {
-        KnowledgeBase prevKnowledgeBase = PluginImpl.getInstance().getKnowledgeBase();
+        PluginImpl instance = PluginImpl.getInstance();
+        KnowledgeBase prevKnowledgeBase = instance.getKnowledgeBase();
         FailureCause cause = new FailureCause("Olle", "Olle");
         cause.addIndication(new BuildLogIndication(".*olle"));
         cause = prevKnowledgeBase.addCause(cause);
         StaplerRequest sreq = mock(StaplerRequest.class);
         DifferentKnowledgeBase knowledgeBase = new DifferentKnowledgeBase("Hello");
-        when(sreq.bindJSON(eq(KnowledgeBase.class), isA(JSONObject.class))).thenReturn(knowledgeBase);
-
-        JSONObject form = createForm("x", PluginImpl.DEFAULT_NR_OF_SCAN_THREADS, true);
-
-        PluginImpl.getInstance().configure(sreq, form);
-
-        assertNotSame(prevKnowledgeBase, PluginImpl.getInstance().getKnowledgeBase());
-        assertSame(knowledgeBase, PluginImpl.getInstance().getKnowledgeBase());
-        assertEquals(1, knowledgeBase.getCauses().size());
-        assertSame(cause, knowledgeBase.getCauses().iterator().next());
-
-        //Check that the config page contains what we expect as well.
-        HtmlPage page = jenkins.createWebClient().goTo("configure");
-        assertConfigPageRendering(knowledgeBase, page);
-    }
-
-    /**
-     * Tests {@link PluginImpl#configure(org.kohsuke.stapler.StaplerRequest, net.sf.json.JSONObject)}. with the same
-     * KnowledgeBase type but different configuration.
-     *
-     * @throws Exception if so.
-     */
-    @Test
-    public void testConfigureConvertSameType() throws Exception {
```

Comment:
Why is this test removed?

Reply:
I "think" it was because this test was testing form binding and form binding is now using databound semantics and tested via the jcasc test, re-testing databinding here would be awkward.

Reply:
Why?
Having the plugin behave as normal when upgrading is essential. And just because the databinding works in JCasC doesn't mean it works for Jelly form binding :)
And it is definately testing in the area where there are changes, so if they doon't pass the something has regressed?

Reply:
Restored the test, this one shouldn't have been removed